pkg/kubelet/dockertools/fake_docker_client.go

```diff
@@ -206,11 +212,26 @@ func (f *FakeDockerClient) StartContainer(id string, hostConfig *docker.HostConf
 			Config:     &docker.Config{Image: "testimage"},
 			HostConfig: hostConfig,
 			State: docker.State{
-				Running: true,
-				Pid:     os.Getpid(),
+				Running:   true,
+				Pid:       os.Getpid(),
+				StartedAt: time.Now(),
 			},
 			NetworkSettings: &docker.NetworkSettings{IPAddress: "1.2.3.4"},
 		}
+		if f.ContainerMap != nil {
```

Comment:
@yujuhong I agree it makes sense to fix things you see are problematic. However at the same time kubelet is already quite complex and so are the tests. I've seen multiple times that a simple mechanical change took hours just because author made a small mistake. Debugging tests for a complex component that one does not understand nor own can be pretty hard.

@gmarek is doing this change in kubelet only to unblock his work on kubemark. I guess that in such scenario fixing all the things around doesn't make much sense.

WDYT?